modules/ocf_backups/manifests/pgsql.pp

```diff
@@ -0,0 +1,14 @@
+class ocf_backups::pgsql {
+  include ocf::packages::postgres
+
+  file {
+    '~/.pgpass':
+      source    => 'puppet:///private/backups/.pgpass',
```

Comment:
this will actually need to be done in the `ocf_postgres` module, as this will add it locally on hal instead of on the remote machine

Reply:
in ocf_postgres should this be linked to '/opt/share/.pgpass' then? since it's the home folder of ocfbackups on lethe

Reply:
in `ocf_postgres/manifests/init.pp` you probably want to add a rule like

```puppet
file {
  '/opt/share/.pgpass': 
    source => 'puppet:///private/pgpass',
    mode   => '0400',
    owner  => 'ocfbackups',
    show_diff => false,
}
```
or something like that, with some explanatory comments